e2e/tests-dfx/metadata.bash

```diff
@@ -166,9 +166,9 @@ teardown() {
   dfx canister metadata prebuilt_custom_blank_build candid:service >from_canister.txt
   diff main.did from_canister.txt
 
-  # this canister has a build step, but it is an empty array, so dfx leaves the candid:service metadata as-is
-  dfx canister metadata prebuilt_custom_empty_build candid:service >from_canister.txt
-  diff main.did from_canister.txt
```

Comment:
Is there a reason to delete this part of this test?  This test is verifying existing behavior.

Reply:
Fixed.